net/davix/src/TDavixFileInternal.h

```diff
@@ -123,19 +101,16 @@ class TDavixFileInternal {
      return replicas;
    }
 
-   TMutex positionLock;
-   TMutex openLock;
-
    std::vector<std::string> replicas;
 
    // DAVIX
    Davix::Context *davixContext;
-   Davix::RequestParams *davixParam;
-   Davix::DavPosix *davixPosix;
-   Davix_fd *davixFd;
+   Davix::RequestParams *davixParam = nullptr;
+   Davix::DavPosix *davixPosix = nullptr;
+   Davix_fd *davixFd = nullptr;
    TUrl fUrl;
    Option_t* opt;
```

Comment:
Is this initialization left unchanged on purpose?

Reply:
Yes, because this data member is initialized in the constructor anyway.